client/dashboard/task-list/tasks/shipping/rates.js

```diff
@@ -8,14 +8,16 @@ import { Button, TextControl } from 'newspack-components';
 import classnames from 'classnames';
 import { Component, Fragment } from '@wordpress/element';
 import { FormToggle } from '@wordpress/components';
-import { get } from 'lodash';
 import PropTypes from 'prop-types';
 
 /**
  * WooCommerce dependencies
  */
 import { Flag, Form } from '@woocommerce/components';
 import { getCurrencyFormatString } from '@woocommerce/currency';
+import { CURRENCY } from '@woocommerce/wc-admin-settings';
+
+const { symbol, symbolPosition } = CURRENCY;
```

Comment:
In the feedback on blocks I mentioned the idea of having a currency prefix/suffix option which contained the symbol in the correct position. This is the type of scanario I was referring to - it makes it easier to render prices if you don't need to conditonally look at left/right/right_space/left_space :)

Reply:
Yup, agreed here also.  Punt to a separate pull?  To ensure we don't forget, do an issue?

Reply:
https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/983